infrastructure/terraform/components/acct/data_iam_policy_document_sso_read_only_table_access.tf

```diff
@@ -0,0 +1,125 @@
+data "aws_iam_policy_document" "sso_read_only_table_access" {
+
+  statement {
+    sid    = "AllowGlueAccess"
+    effect = "Allow"
+
+    actions = [
+      "glue:BatchGetTable",
+      "glue:GetDatabase",
+      "glue:GetDatabases",
+      "glue:GetPartition",
+      "glue:GetPartitions",
+      "glue:GetTable",
+      "glue:GetTables"
+    ]
+
+    resources = concat(
+      local.core_glue_catalog_resources, # Access to all core account catalogs is required as they are all accessible via the default catalog in the environment's account
```

Comment:
Why does this need access to core resources? I'd expect read-only account to be restricted to just this account.

Reply:
What we found when configuring the same sort of access on the gateway instance was when configuring the ODBC connection in PowerBI Desktop, if it couldn't list ALL catalogs it would fail, including those in core as they're reachable via the local catalog.

Reply:
Though PowerBI Desktop requires the ability to list all visible catalogs (including remote), the access is further restricted such that the user can only access the desired tables.